### ISSUE TITLE
[Feature Request] Per-Game config

### ISSUE THREAD

**Issue by PolygonSlayer** [PCSX2/pcsx2#3498]
I wasn't able to find this requested anywhere and I'm aware there are several third party solutions to this (I'm currently using RocketLauncher), but it would be preferable if PCSX2 itself had a way to add per-game configs as well as just a default option that one could set that would apply to any game that hasn't had a custom one made.

Considering there are third party solutions to this I presume this will be low priority, but doesn't hurt to throw a request in :)

**Comment by BroKill** [comment]
+1 for this feature, would be really nice.

I'm using LaunchBox as frontend and I'm able to change game configuration using the "cfgpath" command line but I think this feature in PCSX2 will save a lot of time  :)

Hope some dev here will work on it.

CHEERs

**Comment by RedDevilus** [comment]
Why aren't you using the Launchbox Configurator for launchbox then? Otherwise just use Spectabis. It's low priority to make it part of the main PCSX2.

Here is my guide (needs a new update).
[Pergame configuration(spectabis + plugin) v2.pdf](https://github.com/PCSX2/pcsx2/files/4837936/Pergame.configuration.spectabis.%2B.plugin.v2.pdf)

**Comment by PolygonSlayer** [comment]
As I stated in the original request I would think this is low-priority, but you never know, there might be a dev out there who wouldn't mind taking it on.
The third party solutions work fine, but it won't beat having just be part of the emulator itself. I've used the Per-Game config option so much in RetroArch recently and it's so dead easy to use rather than having to deal with third party software on top of it.

**Comment by RedDevilus** [comment]
Everyone is free to make a PR. I do like it to be added too but doing everything and dividing attention is one way to lose control of the project.

**Comment by BroKill** [comment]
@RedDevilus,

I use another method for LaunchBox without installing nothing and without using 3rd party programs.

I load the game in PCSX2, then I set configuration as I want ( not really, let me rephrase...as the game needs ), then I copy the files form the inis folder, make a new folder insode with the name of the game and copy those inis there.
I re-set the configuration as before so all the games need no special configuration use always the general one.
I load LB and in the game setting for the emulator I add the parameters :
(example)

**--cfgpath="Drive letter\PCSX2\inis\Folder named as the game that needs configuration"**

I add other things like  **--fullscreen** ( for fullscreen ) **--nogui** ( for hiding the PCSX2 GUI at launch ) and **--fullboot** ( to set language I chose in the emulator for games that have no option in-game for changing language ( Multi5 PAL ).

Use the quote only if there are space between names.

Then I launch the game via LB and all is good   :)



CHEERs

**Comment by RedDevilus** [comment]
I know the method: https://wiki.pcsx2.net/Command-line_support. Anyway not much can be done to much point anyway to discuss this further because there are solutions/workarounds for what you want. Otherwise someone does the work itself and does a PR.
Good day.

**Comment by RedDevilus** [comment]
It's also been asked in the past: https://github.com/PCSX2/pcsx2/issues/2551, https://github.com/PCSX2/pcsx2/issues/3068

**Comment by PolygonSlayer** [comment]
Ah, that's fair enough then. If it's already tracked then feel free to close it. I just wasn't able to find it :)

**Comment by tadanokojin** [comment]
closing in favor of #2551
